delta_test.go

```diff
@@ -2075,18 +2082,26 @@ func TestCommitLogStore_UnlimitedConcurrent(t *testing.T) {
 func setUpMultiClusterLogStoreTest(t *testing.T) (logStoreTableName string, firstTable *Table, secondTable *Table, actions []Action, operation Write, appMetaData map[string]any) {
 	t.Helper()
 
+	cfg, _ := config.LoadDefaultConfig(context.TODO(), config.WithRetryer(func() aws.Retryer {
+		return retry.AddWithErrorCodes(retry.AddWithMaxAttempts(retry.NewStandard(func(so *retry.StandardOptions) {
+			so.RateLimiter = ratelimit.NewTokenRateLimit(1000000)
+		}), 1000000), (*types.ProvisionedThroughputExceededException)(nil).ErrorCode())
```

Comment:
I think MaxAttempts 1000000 is going to hang if, for example, the internet connection to S3 is down.  Wasn't there a config option for this instead of hardcoding it?

Reply:
Oh, wait, this is in the test - and the test will time out after some period of time.  Never mind!